### ISSUE TITLE
Tensorboard does not work after updating Tensorflow to version 1.4

### ISSUE THREAD

**Issue by ybsave** [tensorflow/tensorboard#695]
I just updated my Tensorflow to version 1.4 with Tensorboard version 0.4.0rc2. However, the new version of Tensorboard cannot show anything for a log directory which was previously displayed without any problem before this update.

OS: Windows 10
Browser: Google Chrome Version 61.0.3163.100 (Official Build) (64-bit)
`Test results with "inspection" command:

Found event files in:
F:\projects\DeepLearning\TensorFlow\Models\Cifar\densenet_40_12_cifar_10_augmented
F:\projects\DeepLearning\TensorFlow\Models\Cifar\densenet_40_12_cifar_10_augmented\eval

These tags are in F:\projects\DeepLearning\TensorFlow\Models\Cifar\densenet_40_12_cifar_10_augmented:
audio -
histograms -
images
   images/image/0
   images/image/1
   images/image/2
   images/image/3
   images/image/4
   images/image/5
scalars
   cross_entropy_1
   global_step/sec
   learning_rate_1
   reg_loss
   total_loss
   train_accuracy_1
tensor -

Event statistics for F:\projects\DeepLearning\TensorFlow\Models\Cifar\densenet_40_12_cifar_10_augmented:
audio -
graph
   first_step           0
   last_step            0
   max_step             0
   min_step             0
   num_steps            1
   outoforder_steps     []
histograms -
images
   first_step           1
   last_step            99901
   max_step             99901
   min_step             1
   num_steps            1000
   outoforder_steps     []
scalars
   first_step           1
   last_step            99901
   max_step             99901
   min_step             1
   num_steps            1000
   outoforder_steps     []
sessionlog:checkpoint
   first_step           1
   last_step            100000
   max_step             100000
   min_step             1
   num_steps            100
   outoforder_steps     []
sessionlog:start
   outoforder_steps     []
   steps                [1, 4001, 8001, 12001, 16001, 20001, 24001, 28001, 32001, 36001, 40001, 44001, 48001, 52001, 56001, 60001, 64001, 68001, 72001, 76001, 80001, 84001, 88001, 92001, 96001]
sessionlog:stop -
tensor -

These tags are in F:\projects\DeepLearning\TensorFlow\Models\Cifar\densenet_40_12_cifar_10_augmented\eval:
audio -
histograms -
images -
scalars
   accuracy
   loss
tensor -

Event statistics for F:\projects\DeepLearning\TensorFlow\Models\Cifar\densenet_40_12_cifar_10_augmented\eval:
audio -
graph
   first_step           0
   last_step            0
   max_step             0
   min_step             0
   num_steps            1
   outoforder_steps     []
histograms -
images -
scalars
   first_step           4000
   last_step            100000
   max_step             100000
   min_step             4000
   num_steps            25
   outoforder_steps     []
sessionlog:checkpoint -
sessionlog:start -
sessionlog:stop -
tensor -`

**Comment by jart** [comment]
Does it log anything? Anything in the Chrome console log? Can you post a
screenshot?

**Comment by ybsave** [comment]
it doesn't log anything. :(  Just the regular message on the screen:

No dashboards are active for the current data set.

Probable causes:

You haven’t written any data to your event files.
TensorBoard can’t find your event files.
If you’re new to using TensorBoard, and want to find out how to add data and set up your event files, check out the README and perhaps the TensorBoard tutorial.
If you think TensorBoard is configured properly, please see the section of the README devoted to missing data problems and consider filing an issue on GitHub.

Last reload: Thu Nov 02 2017 16:36:02 GMT-0700 (Pacific Daylight Time)
Log directory: F:\projects\DeepLearning\TensorFlow\Models\Cifar\densenet_40_12_cifar_10_augmented

**Comment by ybsave** [comment]
If this issue cannot be solved soon, would you please share me a link for the old version of Tensorboard and some install instructions for ONLY uninstall/install tensorboard? Previously, I always install it from Tensorflow, but I am not familiar about how to change tensorboard only. Thank you.

**Comment by ybsave** [comment]
Update: I downgraded my Tensorflow-GPU to version 1.3; it also downgraded the Tensorboard to version 0.1.8. However, the tensorboard still does not work. The log directory could be displayed correctly on other computers with Tensorflow-GPU 1.3 without updating to 1.4. 

Does this mean that when upgrading to Tensorflow 1.4, the new installation program damaged some basic functions? Thus, even I downgrade Tensorflow again, Tensorboard still does not work!

Is there any viable solutions? Thank you.

**Comment by ybsave** [comment]
Update: I find out a solution. After I run tensorboard in the same drive, e.g., "F:\", everything works fine!

This is definitely a bug in Windows. In previous versions, I can run Tensorboard on other disks, or even across network without any problem. Hope it will be fixed soon. Thank you.

**Comment by jart** [comment]
I'm glad to hear you got things working. Thanks for reaching out and sharing this information with the community.

**Comment by pasikon** [comment]
Yeah, that was the issue for me as well, thanks for sharing

**Comment by obaidcuet** [comment]
@ybsave 
That work for me as well (start tensorboard from the same drive where logdir is)

**Comment by subham1** [comment]
Yeah. I too ran it in the same directory and it worked. Thanks!
